internal/Employee.go

```diff
@@ -0,0 +1,8 @@
+package internal
+
+type Employee struct {
+	ID         string    `json:"id"`
+	FirstName  string    `json:"firstName"`
+	LasName    string    `json:"lasName"`
+	PositionID *Position `json:"position"`
```

Comment:
Check fields from the Go model with model declared in OpenAPI specification

Reply:
I think, maybe it would be better if I changed filed "position" inside employee schema in the documentation. Can I do it?

Reply:
How do you want to change it?

Reply:
PositionID should be ID of position entity, so the type of the filed should be string

Reply:
Okay, i do it, but according to the task, i should set referential type.